services/openshift/scripts/openshift_provision

```diff
@@ -112,6 +112,55 @@ if [ ! -f ${ORIGIN_DIR}/secured.registry ]; then
   oc get route docker-registry -o json  | sed -e 's/\("spec": {\)/\1 "tls": {"termination": "passthrough"},/g' | oc replace -f -
 fi
 
+
+# Provision persistence volumes
+if [ ! -f ${ORIGIN_DIR}/configured.nfs ]; then
+  echo "[INFO] Creating and configuring NFS"
+  mkdir -p /nfsvolumes/pv{01..03}
+  chown -R nfsnobody:nfsnobody /nfsvolumes
+  chmod -R 777 /nfsvolumes
+  echo '' > /etc/exports
+
+  for i in {01..03}
+  do
+    echo "/nfsvolumes/pv${i} *(rw,root_squash)" >> /etc/exports
+  done
+  # To allow pods to write to remote NFS servers
+  setsebool -P virt_use_nfs 1
+
+  # Start and enable nfs
+  systemctl start rpcbind nfs-server
+  systemctl enable rpcbind nfs-server
+
+  # Enable the new exports without bouncing the NFS service
+  exportfs -a
+
+  echo "[INFO] Creating 3 NFS PV {pv01..03} using from 1Gi  3Gi in ReadWriteMany or ReadWriteOnly mode and Recycle Policy."
+  for i in {1..3}
+  do
+  cat <<-EOF > /tmp/pv.yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv0${i}
+spec:
+  capacity:
+    storage: ${i}Gi
+  accessModes:
+    - ReadWriteOnce
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Recycle
+  nfs:
+    server: localhost
+    path: /nfsvolumes/pv0${i}
+EOF
+  # Create 3 volumes from 1Gi to 3Gi
+  oc create -f /tmp/pv.yaml
+  done
+
+  touch ${ORIGIN_DIR}/configured.nfs
+fi
+
```

Comment:
@LalatenduMohanty I am not sure if we have to put cleanup for this because it will remove user data and that is not our intention, what if user want to use it for different service like `kubernetes` because all we are creating some bunch of nfs volumes locally and user mounting it on different pods and those volumes are going to contain user data which might required for later use case.